File: app/src/main/java/ng/max/binger/activities/SplashActivity.java
package ng.max.binger.activities;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import ng.max.binger.R;

public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        Intent intent = new Intent(this, MainActivity.class);
        startActivity(intent);
        finish();
    }
}

File: app/src/main/java/ng/max/binger/activities/SplashActivity.kt
package ng.max.binger.activities

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle

import ng.max.binger.R

class SplashActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val intent = Intent(this, MainActivity::class.java)
        startActivity(intent)
        finish()
    }
}
